src/libraries/System.IO.FileSystem/src/System/IO/FileStatus.Unix.cs

```diff
@@ -186,7 +186,7 @@ internal DateTimeOffset GetCreationTime(ReadOnlySpan<char> path, bool continueOn
             return UnixTimeToDateTimeOffset(_fileStatus.CTime, _fileStatus.CTimeNsec);
         }
 
-        internal void SetCreationTime(string path, DateTimeOffset time)
+        private void SetCreationTime_OtherUnix(string path, DateTimeOffset time)
```

Comment:
This renaming does not make sense, since this is the `Unix` file. Shouldn't the methods in the _new_ file be the ones named with the `_OtherUnix` prefix?

Reply:
The reason I've renamed it is so that SetCreationTime can be called by other code as it's the same name as before (so I don't have to hunt down references to SetCreationTime and change them all, this way the SetCreationTime_OtherUnix method is the unix implementation so that OSX can call it as a fallback without having 2 copies of the code that could get desynced), but OSX needs the unix behaviour as fallback (otherwise I'd just put it in the .Unix file with the same name as before). I'm happy to call it something else, I'm just not sure what. I've made it in the .Unix file so it just calls these methods directly.
Does that make sense? Or should I do something different?

Reply:
I've renamed it to _StandardUnixImpl in the latest commit as the name makes more sense, still happy to change the name if there's something more logical.